Tasks/DeployVisualStudioTestAgent/make.json

```diff
@@ -2,7 +2,7 @@
      "externals": {
          "archivePackages": [
             {
```

Comment:
1) You need to raise a PR against releases/m112  as well
2) bump up the patch version in task.json and task.loc.json
3) after you push this we also need to update additonaldepedency.xml in vso repo